aopy/preproc/__init__.py

```diff
@@ -3,12 +3,57 @@
 from .oculomatic import parse_oculomatic
 from .optitrack import parse_optitrack
 from .. import postproc
-from ..data import save_hdf, load_hdf_group, get_hdf_dictionary
+from ..data import load_ecube_data_chunked, load_ecube_metadata, proc_ecube_data, save_hdf, load_hdf_group, get_hdf_dictionary
 import os
+import h5py
+from glob import glob
 
 '''
 proc_* wrappers
 '''
+def proc_day(data_dir, result_dir, overwrite=False, save_res=True):
+    files = glob(os.path.join(data_dir,'*.bin'))
```

Comment:
Better if files is an input to this function since it will come from database code that isn’t included here.